views/logs.html

```diff
@@ -0,0 +1,22 @@
+<div class="table-wrapper">
```

Comment:
See I'm not really sure I'm a fan of organizing it this way.  I guess since the files are named the same then its easy to find its counterpart, but I still think it makes more sense for each to have its own folder, especially if one portion was complicated enough to warrant multiple JavaScript files.